COCOAPODS.md

```diff
@@ -187,15 +228,367 @@ import cocoapods.subspec_dependency.*
+
+### Add a dependency on a Pod library from a custom Spec repository
+
+You can add dependencies on a Pod library from a custom Spec repository with `pod()` and `specRepos` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the HTTP address to the custom Spec repository using the `url()` inside the `specRepos` block.
+
+2. Specify the name of a Pod library in the `pod()` function.
+
+3. Specify the minimum deployment target version for the Pod library.
+
+   > If you don't specify the minimum deployment target version and a dependency Pod requires a higher deployment target, you may get an error.
+   {:.note}
+
+   <div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+   ```kotlin
+   kotlin {
+       ios()
+
+       cocoapods {
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           ios.deploymentTarget = "13.5"
+
+           specRepos {
+              url("https://github.com/Kotlin/kotlin-cocoapods-spec.git")
+           }
+           pod("example")
+
+       }
+   }
+   ```
+
+   </div>
+
+4. Re-import the project.
+
+> To work correctly with Xcode, you should specify the location of specs at the beginning of your Podfile.
+> For example, `source 'https://github.com/Kotlin/kotlin-cocoapods-spec.git'`
+{:.note} 
+
+To use these dependencies from the Kotlin code, import the packages `cocoapods.<library-name>`.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+import cocoapods.AFNetworking.*
+import cocoapods.SDWebImage.*
+```
+
+</div>
+
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
+
+### Add a dependency on a Pod library with custom C interoperability options
+
+You can add dependencies on a Pod library with custom C interoperability options with `pod()` to `build.gradle.kts` 
+(`build.gradle`) of your project:
+
+1. Specify the name of a Pod library in the `pod()` function.
+In the configuration block specify the C interoperability options:
+
+    * `extraOpts` – to specify the list of options to a Pod library. For example, specific flags
+    * `packageName` – to specify the package name. If you specify it, you can import the library using the package name: `import <packageName>`
+
+2. Specify the minimum deployment target version for the Pod library.
+
+   > If you don't specify the minimum deployment target version and a dependency Pod requires a higher deployment target, you may get an error.
+   {:.note}
+
+   <div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+   ```kotlin
+   kotlin {
+       ios()
+   
+       cocoapods {
+           summary = "CocoaPods test library"
+           homepage = "https://github.com/JetBrains/kotlin"
+
+           ios.deploymentTarget = "13.5"
+
+           pod("new_pod") {
+               extraOpts = listOf("-compiler-option")
+               packageName = "package"
```

Comment:
Please make sure this actually works, and you are able to import from a package named "`package`" in Kotlin.

Reply:
I replace the example with another one.